app/(music)/track/[id]/TrackView.tsx

```diff
@@ -37,12 +38,15 @@ const TrackView = ({
             </p>
           ),
         buttons: (
-          <Link
-            className="btn btn-primary btn-outline w-full"
-            href={`/album/${track.album.id}`}
-          >
-            View Album
-          </Link>
+          <>
+            <PlayButton uris={[track.uri]} />
```

Comment:
Pass the associated album URI as context

Reply:
Added contextUri and replaced uris with offset.